samples/sample_camera/include/camera_render.h

```diff
@@ -0,0 +1,107 @@
+/******************************************************************************\
+Copyright (c) 2005-2018, Intel Corporation
```

Comment:
I think we should adjust copyright for all files

Reply:
addressed in separate patch of this pull request